apps/storefront/src/pages/ForgotPassword/index.tsx

```diff
@@ -169,7 +169,7 @@ export function ForgotPassword({
               <Captcha
                 siteKey={storefrontSiteKey}
                 size="normal"
-                email={emailAddressReset}
+                email={emailReset}
```

Comment:
odd that a `Captcha` needs to know about the new email

Reply:
checked, email is not required, did some refactoring while in there